dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -308,10 +308,15 @@ ERROR_TRANSFORMING_CLASS_CWWJP0055E.useraction=The class cache might be corrupte
 
 #-------------------------------------------------------------------------------
 # 0=property name, 1=property value
-EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property configured in the server.xml file contains an empty name or empty value: name = [{0}], value = [{1}]. The property will be ignored.
+EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property, configured in the server.xml file, contains an empty name or empty value: name = [{0}], value = [{1}]. The property will be ignored.
```

Comment:
per IBM style, avoid future tense

The property will be ignored
--->
This property is ignored

Reply:
slight edit suggestion

 A JPA default property that is configured in the server.xml file contains an empty name or empty value: